source/2017-04-13-how-we-got-here.html.markdown.erb

```diff
@@ -0,0 +1,145 @@
+time consuming and thus not quite cheap, but nearly indestructible and perfect
+for outdoors
+* [Netbox](https://routerboard.com/RB911G-5HPacD-NB) - Also DIY, pretty huge and
+not quite cheap, also not the most aesthetic ones
+* [cAP2n](https://routerboard.com/RBcAP2nD) - Plastic toy, not recommended
+
+### Current AP's
+* [wAP](https://routerboard.com/RBwAPG-5HacT2HnD) - Aesthetic pretty good
+dualband access points with a good range
+* [SXT SA5 ac](https://routerboard.com/RBSXTG-5HPacD-SA) - With an angle of 90°
+perfect for the corners in our coworking space
+* [wAP ac](https://routerboard.com/RBwAPG-5HacT2HnD) - very fast, performant AP
+* [hAP ac](https://routerboard.com/RB962UiGS-5HacT2HnT) - like the wAP ac but with
+a built-in switch
+
+We were also pretty harsh when it comes down to kicking people out of our access points,
+because if you got a bad connection you're giving a bad time to all the other people
+using that access point. When we lowered the connectivity threshold to our sweet spot
+of 65dBm we had an awesome connection, but somehow all the other users were having a
+bad time. This is the point you have to realize that you obviously have another perception
+of "things going well" (although you're right). At the moment we're having 20 AP's on 3
+floors, serving over 300 clients during peak hours.
+
+![ClientsPerAP](ClientsPerAP.jpg)
+
+With all those different devices and total freedom for our coworkers there are
+always some people having connectivity issues nonetheless, so the bigger we get,
+the more important is that we know how to monitor our network and how to distribute
+the load that's hammering on our infrastructure.
+
+![TrafficPerAP](TrafficPerAP.jpg)
+
+## Monitoring
+If you're responsible for something which quite a few people rely on every day, you
+as a sysadmin should be able to tell someone where the problem comes from.
+So it's quite important to monitor your radio waves, so you can start investigating
+rising issues before a lot of people are affected by it and to troubleshoot specific
+clients' issues.
+
+Having some schools and offices around us, we're sometimes experiencing
+a peak in traffic every time there's a bunch of school boys and girls in front of the office
+watching YouTube and whatnot over our free WiFi. Because that free WiFi is actually
+intended for the visitors of the co working Space it's important to drop clients
+that are too far away (across the street inside the school building). That's why
+we're constantly monitoring our frequencies and repositioning our AP's in order
+to supply our guests with a good connection (that's also where our fancy graphs come from).
+
+![SignalStats](SignalStats.jpg)
+
+We're also periodically mapping out our space to check for coverage issues,
+noise sources, interferences and overlapping channels.
+
+![SignalLevel](SignalLevel.jpg)
+
+We went on to develop our own self service portal where you can send us a rating
```

Comment:
I'd still link to it, and mention that it's only available from within the colab

`[our own self service portal (only accessible from within the network)](http://...)`